ssl/t1_trce.c

```diff
@@ -427,6 +427,7 @@ static const ssl_trace_tbl ssl_ciphers_tbl[] = {
     {0xC0AD, "TLS_ECDHE_ECDSA_WITH_AES_256_CCM"},
     {0xC0AE, "TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8"},
     {0xC0AF, "TLS_ECDHE_ECDSA_WITH_AES_256_CCM_8"},
+    {0xC102, "GOST2012-GOST8912-GOST8912"},
```

Comment:
Worth mentioning IANA and legacy code here too.

Reply:
Thanks for pushing the fix!
But could you also please write a note about it when done?